_gsocproposals/2021/proposal_ATLASCompressionAE.md

```diff
@@ -0,0 +1,57 @@
+
+ The aim of this project is to understand the standalone performance of an existing AE network, and whether this network can be used in combination with standard compression methods. Furthermore, a byproduct of this project is that the student will gain expertise in cutting-edge machine learning techniques, and learn to use them in the context of both data compression and detection of anomalous events. 
+
+ It will also be interesting to treat this study as a proof-of-principle for future data compression techniques for the ATLAS experiment. For the planned experimental upgrades in 2026, the techniques used in this report may help solving the problem of needing much more storage space than in the past due to the increase of the size of the dataset. 
+
+ ## Task ideas
+
+    * Run, update and benchmark an existing autoencoder network to compress ATLAS data.
+      * Benchmarking includes and is not necessarily limited to:
+         * The costs of training and decompressing the data in the context of a resource-constrained system such as the ATLAS trigger;
+         * The fidelity of decompressed data with respect to original inputs, in terms of impact on the quality of the physics analysis with de-compressed objects; 
+         * The dependence of the quality of the output on the similarity between training sample and test sample.
+   * Evaluate the performance of combinations of other compression algorithms  
+   * Study the performance of the network as an anomaly detection algorithm, injecting simulated signal data into background data 
+
+ ## Expected results
+
+An improved autoencoder network with documentation and figures of merit that may include: 
+  * plots made in MatPlotLib/ROOT that should demonstrate the performance of the existing network on a different sample of ATLAS data with respect to what was used before for prototyping the network. 
+  * plots of already-compressed variables
+  * plots of the performance on signals that the network hasn't yet been tested on 
+
+ ## Requirements
+
+ Required: Good knowledge of UNIX, Python, MatPlotLib or ROOT, Pytorch, Pandas, fastai, ability to install software and run a Jupyter notebook locally
+
+ ## Evaluation tasks
+
+ Jupyter and pyTorch exercise of running the existing NN and describing the results. 
+
+ ## Links
+  * [ROOT](https://root.cern/)
+  * [Jupyter](http://jupyter.org)
+  * [PyTorch](http://pytorch.org)
+
+ ## Mentors
+  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch) [main]
```

Comment:
The main mentor should be in bold letters

Reply:
thanks, should be fixed